community/rds/aurora-serverless/template.yml

```diff
@@ -0,0 +1,70 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Aurora serverless cluster
+
+Parameters:
+  DatabaseName:
+    Type: String
+  MasterUserPassword:
+    Type: AWS::SSM::Parameter::Value<String>
+    Description: Parameter Store name
+  VpcId:
+    Type: String
+  VpcSecurityGroupId:
+    Type: String
+  BastionKeyName:
+    Type: String
+    Description: EC2 key used to connect to the bastion host
+    Default: Default
+
+Resources:
+  Cluster:
+    Type: AWS::RDS::DBCluster
+    Properties:
+      Engine: aurora
+      EngineMode: serverless
+      EngineVersion: 5.6
+      DatabaseName: !Ref DatabaseName
+      MasterUsername: root
```

Comment:
Again, is it typical to have MasterUserName as "root" or does it make sense to parameterize?